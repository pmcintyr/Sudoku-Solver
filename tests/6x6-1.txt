6 1
2.3...
5...3.
..4...
...5..
.5...3
...4.1
0 1 1 1 1 1
0 0 0 0 0 1
2 3 3 3 3 3
2 2 2 2 2 3
4 5 5 5 5 5
4 4 4 4 4 5
