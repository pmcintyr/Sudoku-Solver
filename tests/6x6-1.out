~~~~~~ Initial ~~~~~~
╔═════════════╗
║ 2 . 3 . . . ║
║ 5 . . . 3 . ║
║ . . 4 . . . ║
║ . . . 5 . . ║
║ . 5 . . . 3 ║
║ . . . 4 . 1 ║
╚═════════════╝
~~~*~~ Solved! ~~*~~~
╔═════════════╗
║ 2 6 3 1 5 4 ║
║ 5 4 1 6 3 2 ║
║ 6 1 4 3 2 5 ║
║ 4 3 2 5 1 6 ║
║ 1 5 6 2 4 3 ║
║ 3 2 5 4 6 1 ║
╚═════════════╝
